charts/invenio/templates/vocabularies-configmap.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.invenio.vocabularies}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: invenio-vocabularies
+  labels:
+    {{- include "invenio.labels" . | nindent 4 }}
+data:
+{{- range $key, $value := .Values.invenio.vocabularies }}
+  {{ $key }}: |
+  {{- $value | nindent 4 }}
+{{- end }}
```

Comment:
Not a bug, I just want to highlight that this logic can probably be replaced by `{{- .Values.invenio.vocabularies | toYaml | nindent 2 }}` or something like that. No need to iterate over the objects.